### PR TITLE
Implement app query logic for mac

### PR DESCRIPTION
**Pull request by nathancharles**
- Add app query logic for mac
- Update UI to show correct messaging
- Mark init and runall methods as deprecated as this will break
- Some formatting changes with standard formatter
- Update graphql to modern version

### CODE REVIEW

File: src/Device.js [hunk 1]
```diff
@@ -38,6 +39,52 @@ class Device extends Component {
...
+    )
+  }
+
+  renderApplicationStateMessage = ({ state, version, installed, policy = {} }) => {
+    if (state === 'INVALID_INSTALL_STATE' && !installed) {
+      const installPrompt = policy.installFrom ? (
+        <p>
+          {' '}
+          Install from{' '}
+          <a href={policy.installFrom} target='_blank'>
```
Comment (rmcvey) [inline src/Device.js]: small nit, should have `rel="noopener noreferrer"` to prevent compilation warning

**Comment by rmcvey** [comment]
Overall this looks great, nice work! I think the app name handling needs a little work, for example:

```query ValidateDevice($policy: DevicePolicy!) {
  policy {
    validate(policy: $policy) {
      status
      applications {
        name
        status
      }
    }
  }
}
```

Policy
```
{
  "policy": {
    "applications": [{
      "name":"BIG-IP Edge Client",
      "version": ">=7180.2019.0118",
      "installed": "ALWAYS"
    }]
  }
}
```

This fails because the spaces aren't escaped. Manually escaping the spaces with `\\` fixes the space parsing issue with `defaults read` but causes `kmd` to fail. The names that are returned in the response seem to have some escape sequences too. I think we probably need to quote the path:

```
tryExec defaults read '%PATH%/%NAME%.app/Contents/Info.plist'
```

We should also support tilde expansion as well, which might require string matching and replacing with  `os.homedir()` - I haven't tested, it might work.

I'm also having some issues with non-standard paths, for instance:

```
{
    "name": "Identity API Scope Approval UI",
    "paths":{
      "darwin": "/Users/rmcvey/Applications/Chrome Apps.localized"
    },
    "version": ">=3683.103",
    "installed":"ALWAYS"
}
```
This fails, regardless of whether or not I quote the path in `app.sh`, but ```defaults read /Users/rmcvey/Applications/Chrome\ Apps.localized/Identity\ API\ Scope\ Approval\ UI.app/Contents/Info.plist``` works fine

**Comment by rmcvey** [comment]
Just to provide an example of the `application.name` issues when the path is quoted: 

```
"applications": [
  {
    "name": "BIG-IP Edge Client\"",
    "status": "PASS"
  },
  {
    "name": "Identity API Scope Approval UI\"",
    "status": "FAIL"
  }
]
```

**Comment by nathancharles** [comment]
Thanks for catching the space issues! Adding the single quote and updating the extract regex worked to fix that. 

Good call on the tilde expansion, `ox.homedir()` worked great!

File: package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "name": "Stethoscope",
   "version": "3.0.6",
   "private": true,
-  "homepage": "./",
+  "homepage": "http://stethoscope.app",
```
Comment (rmcvey) [inline package.json]: small nit - can you switch this to https?

**Comment by rmcvey** [comment]
This is working perfectly now! 

Last request: would you mind reducing the padding between the `<p>`'s - it's a pretty large gap between apps. Other than that, I think this is ready to ship

Stethoscope__v3_0_6_

**Comment by rmcvey** [comment]
One more thing I noticed (which is probably fine for this release) - missing suggested applications correctly pass the validation, but surface to the user as if they are installed. For example:

```
{
  "name": "Foo Bar Baz I Made This Up",
  "paths":{
    "darwin": "~/Applications"
  },
  "installed":"SUGGESTED"
}
```
Stethoscope__v3_0_6_

**Comment by rmcvey** [comment]
Attempting to build the app now and it appears the new `Security.test.js` file is causing `standard` linting to fail, can you add `"**/*.test.js"` to the `standard.ignore` list in `package.json`?

**Pull request merged**